### ISSUE TITLE
1080 works great for .5 - 1.5 hours.  Crashes after

### ISSUE THREAD

**Issue by FanMInc** [OhGodACompany/OhGodAnETHlargementPill#63]
So I'm running two ASUS GTX 1080 STRIX and maintained 40~ MH/s combined using Claymore's v11.7.   Have been stable for around 36 hours straight. 

I decide to Google how to increase hashrate and came across your wonderful product.  Started it as instructed (once miner has started) and works great; taking my hashrate to 60 MH/s combined.  WOW!  Works great for a bit then crashes my video cards.

Any ideas?  I saw where v11.6 might be best for V.2 of the pill.  Also saw where V.1 might be better if you can find it IF V.2 is unstable for you.

**Comment by FanMInc** [comment]
That instantly crashes the cards!!

I found a V.1 of the pill.  Ran for 28 hours straight until it also crashed.  I guess I will keep using V.1 until I find something else.  

I just updated the GPUs to the latest driver (397.64) so maybe that'll help the stability.

**Comment by diehummel** [comment]
My Gainwar Pheonix is also crashing abfter a short time and also if its startet before the miner.
Where did you get the v1 release?

**Comment by diehummel** [comment]
revA keeps also crashing. It just takes a few hours longer until the card gets out of memory.

**Comment by diehummel** [comment]
i found , that when uncommenting
rem setx GPU_FORCE_64BIT_PTR 0
rem setx GPU_MAX_HEAP_SIZE 100
rem setx GPU_USE_SYNC_OBJECTS 1
rem setx GPU_MAX_ALLOC_PERCENT 100
rem setx GPU_SINGLE_ALLOC_PERCENT 100
ethminer.exe ....

in the batch file (windows), it seems now more stable. Running now 33Mhs much longer stable then before with this environment parameters. Even with overclocking.

**Comment by ruminaire** [comment]
@FanMInc what is your setting with ASUS GTX 1080 STRIX?
my 1080 STRIX hashing ~24.5 MH/s without pill at 60% (108W) power limit, and ~27 MH/s with pill. That is kind of low, it won't go past 27 MH/s even with higher power limit 70-75% i wonder why.
running pill before the miner also crashing, only works after miner starts and create DAG.

for reference my other 1080 cards:
zotac 1080 FE is hashing ~36 MH/s after pill at 60% (108W) power limit.
zotac 1080 amp extreme is hashing ~38.5 MH/s after pill at 50% (135W) power limit.

i wonder why my 1080 STRIX is so low, yours is ~30 MH/s right? (60 MH/s with two of them)

**Comment by FanMInc** [comment]
@ruminaire Sorry, just now seeing your question.
Honestly.. I'm not sure.  I haven't overclocked outside of ASUS GPU TweakII software.

I always have it up to monitor temperatures and it stays at 72°C with 56% fan speed, GPU/Memory Clock at Max, and Memory Usage at 3389 MB.  From within Claymore miner v11.7, I'm using the default -dcri values of 30 each.  I haven't messed with these to see if I can get high hashrates.

Using the R.1 version I found, it is still extremely stable (I have no idea how to run the --revA unless that is through the .bat file?).  It went over 100 hours without fault earlier this week.  This is my gaming machine so I turn it off to play PUBG!  

That is correct in both of my 1080 STRIX run at ~30 MH/s compared to 20 MH/s without the Pill.  

For bonus information, the Pill only works for me only after the mining software has fully started and is receiving new ETH jobs.  I created a .bat file for when I'm done playing video games and can walk away from the computer where it starts the miner .bat file (with settings called out by @diehummel) a WAIT command with "ping localhost -n 60 >nul" (waits 60 seconds before continuing) and then start the Pill.exe!  I also have a similar one in the StartUp folder in case the computer crashes while mining.
